cmd/buildah/addcopy.go

```diff
@@ -38,20 +34,13 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 		name = c.String("name")
 	}
 	root := c.String("root")
-	link := ""
-	if c.IsSet("link") {
-		link = c.String("link")
-		if link == "" {
-			return fmt.Errorf("link location can not be empty")
-		}
-	}
 	dest := ""
 	if c.IsSet("dest") {
 		dest = c.String("dest")
 	}
-	if name == "" && root == "" && link == "" {
+	if name == "" && root == "" {
 		if len(args) == 0 {
-			return fmt.Errorf("either a container name or --root or --link, or some combination, must be specified")
+			return fmt.Errorf("either a container name or --root, or some combination, must be specified")
```

Comment:
Are we sticking with container name or container id for our verbiage?  I'd prefer id, but would like to be consistent.  For this message and others similar to it, can we make it:

"either a container name or --root must be specified"